### ISSUE TITLE
Editor: images embeded with no alt or description

### ISSUE THREAD

**Issue by xiaoniuniu89** [codu-code/codu#503]
## Context

This is a story of master feature #197 The editor is in components/editor

## Expected Behavior

when I embed an image I should be prompted for alt text and a description for accessability.

## Current Behavior

user is prompted for url only

## Steps to reproduce

Please provide detailed steps for reproducing the issue

1. go to alpha/create/new
2. click add image from toolbar
3. paste this link, of a .. cat .. obviously https://t4.ftcdn.net/jpg/00/97/58/97/360_F_97589769_t45CqXyzjz0KXwoBZT9PRaWGHRk5hQqQ.jpg

## Additional info

https://tiptap.dev/api/nodes/image

**Comment by AksBad007** [comment]
Hi, if this is still is open, I'd like to be assigned.

**Comment by John-Paul-Larkin** [comment]
Hi niu. I am working on this today. 
When using the alpha/new editor to create a test post, I just get the json data for the post. I will include a short video to show you what I mean.
This is on upstream/develop branch, so nothing I have done. I can still complete the issue, since i can see the alt and title in the json. I am just curious if you know why this is happening. Thanks.

https://youtu.be/t4yBvMLhPNI
